Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -68,6 +72,12 @@ __declspec(dllexport) DWORD NvOptimusEnablement = 1;
 }
 #endif
 
+std::string VideoBackendBase::BadShaderFilename(const char* shader_stage, int counter)
+{
+  return fmt::format("{}bad_{}_{}_{}_{}.txt", File::GetUserPath(D_DUMP_IDX), shader_stage,
+                     Common::scm_rev_str, g_video_backend->GetDisplayName(), counter);
```

Comment:
Are you sure you want the display name here, instead of the name? This would generate filenames containing say, "Direct3D 12" instead of "D3D12".

Reply:
I forgot about `GetName`. Changed.